diff --git a/CMakeLists.txt b/CMakeLists.txt
index d50dd09..999b285 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.1.0)
-cmake_policy(SET CMP0043 OLD)
+cmake_policy(SET CMP0043 NEW)
 
 include(FindPkgConfig)
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules)
@@ -56,4 +56,4 @@ target_link_libraries(perseus-sdr
 )
 
 install(FILES perseusfx2.h  perseus-in.h  perseus-sdr.h DESTINATION include)
-install(TARGETS perseus-sdr DESTINATION ${LIB_INSTALL_DIR})
\ No newline at end of file
+install(TARGETS perseus-sdr DESTINATION ${LIB_INSTALL_DIR})
